extras/nginx/nginx-1.16.patch

```diff
@@ -4035,6 +4035,13 @@ index 000000000..30955c4cd
 +      offsetof(ngx_http_v3_srv_conf_t, idle_timeout),
 +      NULL },
 +
++    { ngx_string("http3_cc_algorithm"),
++     NGX_HTTP_MAIN_CONF|NGX_HTTP_SRV_CONF|NGX_CONF_TAKE1,
++     ngx_conf_set_num_slot,
```

Comment:
Instead of using the CC algorithm IDs directly, it's probably better to add the `set_cc_algorithm_name()` function to the FFI API (https://github.com/cloudflare/quiche/blob/master/src/lib.rs#L894), and then use a string for this configuration value.